findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java

```diff
@@ -268,11 +272,25 @@
         public static final @SpecialKind
         int TYPE_ONLY = 24;
 
+        /**
+         * @deprecated will be deleted at v4 release. use @{link {@link #getSpecialKindName(int)}} instead.
+         */
+        @Deprecated
         @edu.umd.cs.findbugs.internalAnnotations.StaticConstant
         public static final HashMap<Integer, String> specialKindNames = new HashMap<Integer, String>();
```

Comment:
we should probably leave a `TODO` for ourselves to convert this into a `private ConcurrentMap`